app/scripts/modules/core/src/plugins/pluginSupport.ts

```diff
@@ -0,0 +1,38 @@
+import * as lodash from 'lodash';
+import * as react from 'react';
+import * as react_dom from 'react-dom';
```

Comment:
I'm curious about the expected behavior for the bundler and bundle size here. Will each lib in full be put into the bundle so that it is accessible to dynamically loaded plugins via the global object?

Reply:
yes, these whitelisted libraries will be bundled in full into one of the initial deck bundles (which are loaded immediately on startup).  Then they're exposed on the global object for use by plugins.